cgo_enabled.mk

```
@@ -0,0 +1,32 @@
+#
+#  SPDX-License-Identifier: AGPL-3.0-only
+#  Copyright (c) 2022-2024, daeuniverse Organization <dae@v2raya.org>
+#
+
+ifneq ($(KEEP_CGO_ENVS),1)
+    ifdef CGO_ENABLED_NDEF
+        export CGO_ENABLED := 0
+    endif
+    ifeq ($(CGO_ENABLED),0)
+        ifdef GOARCH_NDEF
+            ifeq ($(CC),cc)
+                ifneq ($(shell which musl-gcc),)
```

Using `command -v` is better than `which` when checking the existence of executable, as it is a UNIX-compatible built-in command.
Reference: https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script

```suggestion
                ifneq ($(shell command -v musl-gcc),)
```